README.md

```diff
@@ -12,6 +12,8 @@
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite.svg?type=small)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite?ref=badge_small)
 
+[![Open Source Helpers](https://www.codetriage.com/techqueria/website/badges/users.svg)](https://www.codetriage.com/techqueria/website)
```

Comment:
Can we have this added above FOSSA status instead of after? Looks good otherwise!

Reply:
Sure, will make the changes and push it

Reply:
I have resolved the placement issue and pushed the commit to the same PR